frontend/src/views/web/landing.html

```diff
@@ -10,6 +10,11 @@
 <section class="ev-container center">
     <h4 class="ev-head w-300 text-white">EvalAI</h4>
     <p class="text-light-gray ev-intro w-300">Be a leader in Computational Research</p>
+    <br>
+    <a ng-click="main.hostChallenge($event)" class="waves-effect waves-dark btn ev-btn-dark w-300">Host Challenge&nbsp; <i class="fa fa-paper-plane-o text-light-gray icon"></i></a>
+    <a ui-sref="web.challenge-main" class="waves-effect waves-dark btn ev-btn-light text-dark-black w-300">Participate in Challenges&nbsp; <i class="fa fa-free-code-camp text-dark-black icon"></i></a>
+    <h4 class="ev-head w-300 text-white">EvalAI</h4>
```

Comment:
Please check line line 11 and 16, again replication. Try to check all things in your local server.